CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.5 (2020-04-22)
+
+- Fixed the `skyAutonumeric` directive to properly validate on input changes. [#22](https://github.com/blackbaud/skyux-autonumeric/pull/22)
```

Comment:
We could probably leave out "properly," but I don't have strong feelings about keeping it in either.

Reply:
Done